### ISSUE TITLE
Create weather detail layout

### ISSUE THREAD

**Issue by appdevgenie** [myconceptlife/aad-team-54-animation-challenge#8]
User can click on item in recyclerView for more detail on specific day

**Comment by appdevgenie** [comment]
basic layout and activity complete

**Comment by josephkandi** [comment]
@appdevgenie When you are done, please create a PR before closing the issue. The issue will be closed after reviewing the PR and when its merge into the master branch

**Comment by appdevgenie** [comment]
cool, getting ahead of myself :) ... not so well versed with git, thanks for the guidance

**Comment by josephkandi** [comment]
@appdevgenie I see you are still merging directly to the master branch, please create PR request first, someone needs to review the code before we can merge to master
